File: backend/src/main/java/com/ssafy/zip/entity/Notification.java
package com.ssafy.zip.entity;

import lombok.*;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity
@Getter
@Setter
@AllArgsConstructor
@RequiredArgsConstructor
@Table(name = "notification")
public class Notification{
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long id;

    Long userId;

    String message;

    String link;

    String image;

    Boolean isRead;

    LocalDateTime reg;
}

File: android/app/src/main/java/com/ssafy/zip/android/data/Notification.kt
package com.ssafy.zip.android.data

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.util.Date

@Parcelize
data class Notification (
    val id: Long,
    val message : String?,
    val link : String?,
    val image: String?,
    val isRead: Boolean,
    val reg : Date
) : Parcelable ;